### PR TITLE
Update documentation for Windows development

### PR DESCRIPTION
**Pull request by sethdw**
I wrote down a couple of things as I was setting up the development environment on my Windows machine.

Let me know if my commits are too small - we're quite zealous with "only one change per commit" at my work!

### CODE REVIEW

File: DEVELOPMENT_SETUP.md
Comment (SleepAPI) [inline DEVELOPMENT_SETUP.md:20]: Have you tried `nvm-windows`? I have some concern that recommending setting up WSL in order to develop for Sleep API on Windows might be a tough barrier of entry for users without WSL.

**Review by SleepAPI — COMMENTED** [review]
Just waiting for decision regarding squash vs rebase merge in discord dev channel before approve. Left a comment regarding nvm on Windows. Looks good to me beyond that. Great stuff, thanks for doing this 👍

File: DEVELOPMENT_SETUP.md
Comment (sethdw) [reply DEVELOPMENT_SETUP.md:20]: I haven't, but honestly I still wouldn't recommend it. Once WSL is set up it really does "just work" and makes other setup far easier - especially the Docker setup later.

Comment (SleepAPI) [reply DEVELOPMENT_SETUP.md:20]: Perhaps we could phrase it as "it is recommend to use nvm insice WSL, but other alternatives like nvm-windows exist. Current phrasing makes it seem like nvm-windows is not a good option which I'm not sure of. If people are turned off by the WSL then perhaps they should give nvm-windows a shot rather than giving up.

After that I would personally prefer slightly bigger commits with the reasoning being that it makes the develop/main branch history more parseable if we can just scroll through some slogans of feature commits. Small commits are great for reviewing though. I'm open to suggestions and discussion as always.

**Review by SleepAPI — APPROVED** [review]

**Pull request merged**
